adding_species/contributing.ipynb

```diff
@@ -0,0 +1,990 @@
+   "id": "labeled-overview",
+   "metadata": {},
+   "source": [
+    "and now we will put it all together to create the genome object"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "structural-controversy",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "_genome = stdpopsim.Genome.from_data(\n",
+    "    genome_data.data,\n",
+    "    recombination_rate=_recombination_rate,\n",
+    "    mutation_rate=_mutation_rate,\n",
+    "    citations=[\n",
+    "        _SharakhovaEtAl,\n",
+    "        _Ag1000G,\n",
+    "        _PombiEtAl,\n",
+    "    ],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "hourly-receptor",
+   "metadata": {},
+   "source": [
+    "Finally I'm going to edit the species definition in `species.py` to include a default population size and generation time. \n",
+    "\n",
+    "For this we will again rely on the Ag1000G paper and call the generation time 11 gens per year and the population size approximately 6 million, which corresponds to the Ghana population sample"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "executed-transformation",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "_species = stdpopsim.Species(\n",
+    "    id=\"AnoGam\",\n",
+    "    ensembl_id=\"anopheles_gambiae\",\n",
+    "    name=\"Anopheles gambiae\",\n",
+    "    common_name=\"Anopheles gambiae\",\n",
+    "    genome=_genome,\n",
+    "    generation_time=1/11,\n",
+    "    population_size=6e6, #Ghana population\n",
+    "    citations=[_Ag1000G]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "academic-reviewer",
+   "metadata": {},
+   "source": [
+    "now we can make sure this works at the command line\n",
+    "```\n",
+    "python -m stdpopsim AnoGam --help\n",
+    "```\n",
+    "returns for me\n",
+    "```\n",
+    "usage: __main__.py AnoGam [-h] [--help-models [HELP_MODELS]] [-b BIBTEX_FILE] [-D] [-c] [-L LENGTH] [-i INCLUSION_MASK]\n",
+    "                          [-e EXCLUSION_MASK] [-l LENGTH_MULTIPLIER] [-s SEED] [-d] [-o OUTPUT]\n",
+    "                          samples [samples ...]\n",
+    "\n",
+    "Run simulations for Anopheles gambiae using up-to-date genome\n",
+    "information, genetic maps and simulation models from the literature.\n",
+    "NOTE: By default, the tskit '.trees' binary file is written to\n",
+    "stdout,so you should either redirect this to a file or use the '--\n",
+    "output' option to specify a filename.\n",
+    "\n",
+    "Default population parameters for Anopheles gambiae:\n",
+    "Generation time: 0.09090909090909091\n",
+    "Population size: 6000000.0\n",
+    "Mutation rate: 5.49e-09\n",
+    "Recombination rate: 1.044e-08\n",
+    "\n",
+    "...\n",
+    "...\n",
+    "...\n",
+    "```"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "dated-louis",
+   "metadata": {},
+   "source": [
```

Comment:
We should run the species tests first, as that checks for a bunch of common problems:
```
python -m pytest tests/test_AnoGam.py
```

Could be worth deliberately leaving in an error so that you can show the tests picking it up and then fixing it.